client/post-editor/editor-ground-control/index.jsx

```diff
@@ -143,7 +149,7 @@ export class EditorGroundControl extends React.Component {
 	}
 
 	getCloseButtonPath() {
-		const editorPathRegex = /^(\/block-editor)?\/(post|page|(edit\/[^\/]+))(\/|$)/i;
+		const editorPathRegex = /^(\/block-editor)?\/(post|page|(edit\/[^/]+))(\/|$)/i;
```

Comment:
Is this change related to the label change?

Reply:
Not at all. It was just the linter yelling at me (pre-commit).